.github/workflows/crackmapexec-test.yml

```diff
@@ -1,11 +1,8 @@
 name: CrackMapExec Tests
 
 on:
-  workflow_dispatch:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+  pull_request_review:
+    types: [submitted]
```

Comment:
So this only runs on a PR creation and request for review?

Reply:
This should only start running after you submitted a review. A review is required before merging anyway and this looked like a good trigger for the tests

Reply:
Great!